Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -313,7 +313,9 @@ std::vector<RedumpVerifier::PotentialMatch> RedumpVerifier::ScanDatfile(const st
     // so show a panic alert rather than just using ERROR_LOG
 
     // i18n: "Serial" refers to serial numbers, e.g. RVL-RSBE-USA
-    PanicAlertT("Serial and/or version data is missing from %s", GetPathForSystem(system).c_str());
+    PanicAlertT("Serial and/or version data is missing from %s\n"
+                "Please append serial,version to the datfile URL when downloading",
```

Comment:
Maybe:

```suggestion
                "Please append "%s" (without the quotes) to the datfile URL when downloading",
```

passing `serial,version` as a format argument to make sure translators cannot mess up the string? I think adding an example might be a good idea as well.

Reply:
Done.